tokio/src/sync/mutex.rs

```diff
@@ -373,6 +372,18 @@ where
     }
 }
 
+impl<T> std::fmt::Debug for Mutex<T>
+where
+    T: std::fmt::Debug,
+{
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self.try_lock() {
+            Ok(inner) => f.debug_struct("Mutex").field("inner", &*inner).finish(),
+            Err(_) => f.debug_struct("Mutex").field("inner", &"<locked>").finish(),
```

Comment:
I believe that just using a string constant here will format the string using its' `Debug` impl, which will include quotation marks. The standard library's `Mutex` [uses a dummy struct implementing `Display`](https://doc.rust-lang.org/stable/src/std/sync/mutex.rs.html#397-416) to ensure that it _doesn't_ have quotation marks, so we might want to do something similar. I think the simplest thing to do is just:
```suggestion
            Err(_) => f.debug_struct("Mutex").field("inner", &format_args!("<locked>")).finish(),
```